bin/cron/backup_s3_buckets_to_secondary_account

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'aws-sdk'
+require 'cdo/cron_helper'
+require 'cdo/only_one'
+
+TASK_NAME = 'Backup S3 To Secondary Account'
+DEBUG_OUTPUT = false
+
+BACKUP_BUCKET_ROOT = 'cdo-secondary-backup'
+S3S3MIRROR_DIR = "/home/ubuntu/#{Rails.env}/tools/s3s3mirror/"
+S3S3MIRROR_COMMAND = 's3s3mirror.sh'
+
+BUCKETS_TO_BACKUP = [
+  'audio.code.org',
+  'cdo-animation-library',
+  # 'cdo-art', # 120M elements, 7.1tb, seems to be safe to skip
+  'cdo-backups',
+  'cdo-build-logs',
+  'cdo-build-package',
+  'cdo-circle-utils',
+  'cdo-curriculum',
+  # 'cdo-dist', # 170k, 31gb, unnecessary
+  # 'cdo-dynamo-backup', # 81k, 15gb
+  'cdo-email-images',
+  'cdo-error',
+  'cdo-fetch', # 10k, 9gb
+  'cdo-form-uploads', # 10k, 1.2gb
+  'cdo-lib',
+  # 'cdo-logs', # 20M, 12tb, skip!
+  'cdo-map-reduce',
+  'cdo-offline-2',
+  'cdo-offline-hoc',
+  'cdo-offline',
+  'cdo-repo',
+  'cdo-temp', # 35k, 2mb
+  'cdo-travisci',
+  'cdo-tts', # 2.5k, 500mb
+  'cdo-v3-animations', # 4k, 190mb
+  'cdo-v3-assets', # 455k, 250gb
+  'cdo-v3-files',
+  'cdo-v3-prod-firebase-backups',
+  'cdo-v3-sources-contributors',
+  'cdo-v3-sources', # 5M, 30gb
+  'cdo-videos',
+  'cf-templates-p9nfb0gyyrpf-us-east-1',
+  'codepipeline-us-east-1-418960518203',
+  # 'cucumber-logs', # 1.5M, 80gb
+  'deprecated-data',
+  'dms-cwnluin4utwebpwk33pew5qnem',
+  'downloads.code.org',
+  'hoc-static-all',
+  'hoc-static',
+  'images.code.org',
+  'learn.dev-code',
+  'offline-apps',
+  # 'pd-workshop-surveys',
+  'redshift-tableau', # 25k, 65gb
+  'tableau-connector',
+  'videos.code.org' # 714, 30gb
+]
+
+def debug_puts(text)
+  puts text if DEBUG_OUTPUT
+end
+
+def main
+  unless only_one_running?(__FILE__)
+    CronHelper.report_cron_daily_result TASK_NAME, false
+    HipChat.log 'S3 backup task failed, previous task is still running!', color: 'red', notify: 1
+    return
+  end
+  start_time = Time.now
+  debug_puts "Beginning S3 backup sync at #{Time.now}"
+  HipChat.log("Beginning S3 backup sync at #{Time.now}")
+
+  # s3s3mirror does not function properly if you're not in its root directory
+  Dir.chdir(S3S3MIRROR_DIR)
+  BUCKETS_TO_BACKUP.each do |bucket_name|
+    debug_puts "Backing up #{bucket_name}"
+    debug_puts "Running command: #{S3S3MIRROR_DIR}#{S3S3MIRROR_COMMAND} -C #{bucket_name} #{BACKUP_BUCKET_ROOT}/#{bucket_name}/"
+    # -C marks this as a cross-account copy
+    result = `#{S3S3MIRROR_DIR}#{S3S3MIRROR_COMMAND} -C #{bucket_name} #{BACKUP_BUCKET_ROOT}/#{bucket_name}/`
```

Comment:
Any reason this can't be rewritten as
```
command = "#{S3S3MIRROR_DIR}#{S3S3MIRROR_COMMAND} -C #{bucket_name} #{BACKUP_BUCKET_ROOT}/#{bucket_name}/"
debug_puts "Running command: #{command}"
result = `#{command}`
```